pipeline/brew/brew.go

```diff
@@ -186,6 +177,12 @@ func dataFor(
 	} else {
 		description = *rep.Description
 	}
+	if ctx.Config.Brew.Install != "" {
+		installCmds = ctx.Config.Brew.Install
+	} else {
+		installCmds = "bin.install \"" + ctx.Config.Build.BinaryName + "\""
```

Comment:
there is a [`defaults` pipe](https://github.com/goreleaser/goreleaser/blob/master/pipeline/defaults/defaults.go) that is used to set defaults, can you add it there?